resources/react/README.md

```diff
@@ -55,45 +55,45 @@ Cheat sheets save you time and energy by giving you essential syntax at your fin
+<!-- ### npm Intellisense
+<LevelWithButton :image-right=false desc="Visual Studio Code plugin that autocompletes npm modules in import statements" image="https://christian-kohler.gallerycdn.vsassets.io/extensions/christian-kohler/npm-intellisense/1.3.0/1487797010221/Microsoft.VisualStudio.Services.Icons.Default" link="https://marketplace.visualstudio.com/items?itemName=christian-kohler.npm-intellisense" button="Check it out!" /> -->
 
 ### ES7 React/Redux/React-Native Snippets
-<LevelWithButton desc="Simple extensions for React, Redux and Graphql in JS/TS with ES7 syntax" image="https://dsznajder.gallerycdn.vsassets.io/extensions/dsznajder/es7-react-js-snippets/2.7.1/1585251666137/Microsoft.VisualStudio.Services.Icons.Default" link="https://marketplace.visualstudio.com/items?itemName=dsznajder.es7-react-js-snippets" button="Check it out!" />
+<LevelWithButton :image-right=false desc="Simple extensions for React, Redux and Graphql in JS/TS with ES7 syntax" image="https://dsznajder.gallerycdn.vsassets.io/extensions/dsznajder/es7-react-js-snippets/2.7.1/1585251666137/Microsoft.VisualStudio.Services.Icons.Default" link="https://marketplace.visualstudio.com/items?itemName=dsznajder.es7-react-js-snippets" button="Check it out!" />
 
 ## Useful Resources
-### Awesome React
+### Awesome resourses from React ecosystem
 <LevelWithButton desc="A collection of awesome things regarding the React ecosystem." image="https://github.com/sindresorhus/awesome/raw/master/media/logo.png" link="https://github.com/enaqx/awesome-react" button="Check it out!" />
 
 ### Official GitHub
 <LevelWithButton :image-right=false image="/icons/github.svg" link="https://github.com/facebook/react" button="Check it out!" desc="The official Reactjs github repository"></LevelWithButton>
 
 ### Official Documentation
-<LevelWithButton image="https://upload.wikimedia.org/wikipedia/commons/a/a7/React-icon.svg" link="https://reactjs.org/" button="Read Docs" desc="The official Reactjs documentation can help you..."></LevelWithButton>
+<LevelWithButton image="https://upload.wikimedia.org/wikipedia/commons/a/a7/React-icon.svg" link="https://reactjs.org/" button="Read Docs" desc="The official Reactjs documentation can help you understand important concepts and refer to the API"></LevelWithButton>
 
 ## Further Steps
 We at the Hacklab believe in learning by doing. So here are some ideas for a few projects that you can work on to hone your skills.
 
 ### Create a simple application
 
-#### Todo Apps
+#### Todo App
 <Level desc="This is a simple Todo App that lets a user create, read, update and delete to-do tasks. In the application user can also categories the tasks as shown in the image" image="https://miro.medium.com/max/2848/1*bubW56HD3dK44Uj2Jvs-UA.png"></Level>
 
 #### SnapShot
 <Level desc="This application searches and displays the pictures that the user searches for." image="https://github.com/Yog9/SnapShot/raw/master/snapscout.png"></Level>
 
-#### Giphy
+#### GIFs
```

Comment:
I meant for you to replace the giphs* you had written down there. You can keep the app name as Giphy clone